src/coreclr/debug/di/shimstackwalk.cpp

```diff
@@ -988,6 +989,18 @@ CorDebugInternalFrameType ShimStackWalk::GetInternalFrameType(ICorDebugInternalF
 
 void ShimStackWalk::AppendFrame(ICorDebugFrame * pFrame, StackWalkInfo * pStackWalkInfo)
 {
+    // Subtracting 1 from IP of the leaf frame after an exception to ensure that the source info points at the call site 
```

Comment:
Instead of saying subtract one, I'd say "we've detected we're in a stackwalk where we have an exception and no further managed frames are on top of this frame. To ensure our IP points to the user line that threw the exception, we ask the frame to adjust the IP to the call instruction as currently it points to the instruction after it."

Reply:
Thanks @hoyosjs! Done!